src/panels/config/integrations/integration-panels/ozw/dialog-ozw-refresh-node.ts

```diff
@@ -0,0 +1,292 @@
+
+  @internalProperty() private _active = false;
+
+  @internalProperty() private _complete = false;
+
+  @internalProperty() private _progress = [
+    "ProtocolInfo",
+    "Probe",
+    "WakeUp",
+    "ManufacturerSpecific1",
+    "NodeInfo",
+    "NodePlusInfo",
+    "ManufacturerSpecific2",
+    "Versions",
+    "Instances",
+    "Static",
+    "CacheLoad",
+    "Associations",
+    "Neighbors",
+    "Session",
+    "Dynamic",
+    "Configuration",
+    "Complete",
+  ];
+
+  private _addDevicesTimeoutHandle: any = undefined;
+
+  private _subscribed?: Promise<() => Promise<void>>;
+
+  public disconnectedCallback(): void {
+    super.disconnectedCallback();
+    this._unsubscribe();
+  }
+
+  protected updated(changedProperties: PropertyValues): void {
+    super.update(changedProperties);
+    if (changedProperties.has("_device")) {
+      this._fetchData(this._device);
+    }
+  }
+
+  private async _fetchData(device) {
+    const identifiers: OZWNodeIdentifiers | false = getIdentifiersFromDevice(
+      device
+    );
+    if (!identifiers) return;
+    this.ozw_instance = identifiers.ozw_instance;
+    this.node_id = identifiers.node_id;
+
+    const metaDataResponse = await fetchOZWNodeMetadata(
+      this.hass,
+      this.ozw_instance,
+      this.node_id
+    );
+
+    this._nodeMetaData = metaDataResponse.metadata;
+  }
+
+  public async showDialog(params: OZWRefreshNodeDialogParams): Promise<void> {
+    this._device = params.device;
+  }
+
+  protected render(): TemplateResult {
+    if (!this._device) {
+      return html``;
+    }
+
+    return html`
+      <ha-dialog
+        open
+        hideActions
+        @closing="${this._close}"
+        .heading=${createCloseHeading(
+          this.hass,
+          this.hass.localize("ui.panel.config.ozw.refresh_node.title")
+        )}
+      >
+        ${this._complete
+          ? html`
+              <p>
+                ${this.hass.localize(
+                  "ui.panel.config.ozw.refresh_node.complete"
+                )}
+              </p>
+              <mwc-button @click=${this._close}>
+                ${this.hass.localize("ui.common.close")}
+              </mwc-button>
+            `
+          : html`
+              ${this._active
+                ? html`
+                    <div class="flex-container">
+                      <ha-circular-progress active></ha-circular-progress>
+                      <div>
+                        <p>
+                          <b>
+                            ${this.hass.localize(
+                              "ui.panel.config.ozw.refresh_node.refreshing_description"
+                            )}
+                          </b>
+                        </p>
+                        <p>
+                          ${this.hass.localize(
+                            "ui.panel.config.ozw.refresh_node.node_status"
+                          )}:
+                          ${this._node!.node_query_stage}
```

Comment:
`_node` could be undefined, as it is only defined after the first update comes in.